MediaWikiTelegramBot.js

```diff
@@ -1,20 +1,26 @@
+// Require Telegram API & http request
 var TelegramBot = require( 'node-telegram-bot-api' );
 var request = require( 'request' ).defaults( {
     jar: true
 } );
+
+// Require yaml fs reader
 var yaml = require( 'js-yaml' );
 var fs = require( 'fs' );
 
+// So we can use the other functions
```

Comment:
Well, since you mention it, you could also mention what other functions :)

But you can also simply skip it, because the name "MediaWikiAPI" is pretty self-explanatory..